src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -545,8 +545,6 @@ public static void reload() throws IOException {
         if (Minecraft.getMinecraft().theWorld != null) {
             Iris.getPipelineManager().preparePipeline(Iris.getCurrentDimension());
         }
-        // Reload Sodium/Invalidate Chunk Cache
-        Minecraft.getMinecraft().renderGlobal.loadRenderers();
```

Comment:
Why this change?

Reply:
from
https://github.com/IrisShaders/Iris/blob/6c880cd377d97ffd5de648ba4dfac7ea88897b4f/src/main/java/net/coderbot/iris/Iris.java#L611
https://github.com/IrisShaders/Iris/blob/6c880cd377d97ffd5de648ba4dfac7ea88897b4f/src/main/java/net/coderbot/iris/pipeline/PipelineManager.java#L40